### DISCUSSION TITLE
Leitura de Arquivos. Quem usa atualmente?

### DISCUSSION THREAD

**Discussion by orochasamuel** [orochasamuel/fiscalbr-net#83] [Q&A]
Galera,

Quero saber quem está usando a leitura de arquivos de forma ativa atualmente para saber o que precisa ser melhorado.

@marcosgerene @marcianobandeira @adrianotrentim @ebitencourt @ThalyaAutocom @PGama-Rodrigo @hbdbim @EderWillian @eduarda80

**Comment by hbdbim** [comment]
Eu eventualmente utilizo mas não tenho nenuma observação.

**Reply by marcianobandeira** [reply]
Nunca usei, talvez possa ser interessante para sistemas contábeis

**Reply by orochasamuel** [reply]
> Nunca usei, talvez possa ser interessante para sistemas contábeis

@marcianobandeira sei que é bastante usado por sistemas contábeis.

A Conta Azul por exemplo, exporta um arquivo da ECD (Escrituração Contábil Digital) para importação dos lançamentos contábeis no Calima (sistema contábil que utilizo), eles tem uma lista enorme de layouts suportados.

**Comment by marcosgerene** [comment]
Eu fiz a leitura, mas só usei à época para adicionar bloco H em alguns arquivos prontos.

Desde então não houve mais necessidade aqui.

**Comment by elieltonbitencourt** [comment]
Leitura não usei ainda

**Comment by ThalyaAutocom** [comment]
A leitura dos arquivos não utilizamos por aqui

**Comment by WandersonLSoft** [comment]
Bom dia a todos. 

Eu implementei para unificar os arquivos do sped contribuições.
Pois temos um cliente que preferiu que sua  filial tenha sua base de dados própria e não depender de internet.

Com isso os speds contribuições são gerados independente e depois fazemos a unificação do arquivo com o arquivo da matriz.

Utilizando a dll para fazer a leitura, consegui unificar de forma bem prática.
